smt/pkg/smt/witness_utils.go

```diff
@@ -0,0 +1,11 @@
+package smt
+
+import "fmt"
+
+func intArrayToString(a []int) string {
```

Comment:
?

Reply:
it was a function in witness.go and I just moved it here.

Reply:
okay, just felt weird, there is probably one liners for this.